src/less/button/base/button.less

```diff
@@ -171,6 +204,38 @@ a.fake-btn--secondary[href] {
         .color-token(btn-secondary-active-foreground-color, color-b6);
     }
 }
+
+button.btn--secondary.btn--destructive,
+a.fake-btn--secondary.fake-btn--destructive {
+    .background-color-token(btn-secondary-destructive-background-color, color-background-default);
+    .border-color-token(btn-secondary-destructive-border-color, color-destructive);
+    .color-token(btn-secondary-destructive-foreground-color, color-destructive);
+}
+
+button.btn--secondary.btn--destructive[disabled] svg.icon {
+    fill: currentColor;
```

Comment:
Why is this needed? I think current color should be coming from icon?

Reply:
I believe it was necessary, at least at the point at which I was seeing an issue where the color was off. I just removed it and it looks like it works without it. I'll remove it since it appears to be redundant now.